include/items.php

```diff
@@ -763,6 +764,17 @@ function item_store($arr,$force_parent = false, $notify = false, $dontcache = fa
 		return 0;
 	}
 
+	// Check for already added items.
+	// There is a timing issue here that sometimes creates double postings.
+	// An unique index would help - but the limitations of MySQL (maximum size of index values) prevent this.
```

Comment:
I'm on utf8mb4 on all tables and fields with one of my machines. Since we can't rely that admins will change the collation manually we need a solution for this.